dev.ipynb

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
Not sure if you intended to include this, but example in `.py` format or some test would be easier to understand I think